src/components/Tabs/Tabs.tsx

```diff
@@ -65,6 +74,110 @@ const TabsComponent = ({
 
   const withGaps = mode === "accent" || mode === "secondary";
 
+  function getTabEls(): HTMLDivElement[] {
+    if (!tabsRef.current) {
+      return [];
+    }
+
+    return Array.from(
+      // eslint-disable-next-line
+      tabsRef.current.querySelectorAll<HTMLDivElement>(
+        "[role=tab]:not([disabled])"
+      )
+    );
+  }
+
+  function handleDocumentKeydown(event: KeyboardEvent) {
```

Comment:
```suggestion
  const handleDocumentKeydown = (event: KeyboardEvent) => {
```

и тут на стрелочную функцию 🙏